regression-test/suites/query_p0/sort/sort.groovy

```diff
@@ -23,4 +23,22 @@ suite("sort") {
     qt_sort_string_single_column """ select * from ( select '汇总' as a union all select '2022-01-01' as a ) a order by 1 """
     qt_sort_string_multiple_columns """ select * from ( select '汇总' as a,1 as b union all select '2022-01-01' as a,1 as b ) a order by 1,2 """
     qt_sort_string_on_fe """ select '汇总' > '2022-01-01' """
+
+    sql """CREATE TABLE IF NOT EXISTS NO2PhaseSortWhenAggTable
```

Comment:
`2PhaseSortWhenAggTable` could be more correct

Reply:
Oh, I'll change it soon.